pubspec.yaml

```diff
@@ -1,5 +1,5 @@
 name: source_maps
-version: 0.10.11-dev
+version: 0.10.11
 description: Library to programmatically manipulate source map files.
 repository: https://github.com/dart-lang/source_maps
 
```

Comment:
👇  While you're at it, bump the min SDK to 2.18 – then use lints v2!!

Reply:
done